src/server/pfs/server/api_server.go

```diff
@@ -261,13 +265,23 @@ func (a *apiServer) ListCommit(request *pfs.ListCommitRequest, respServer pfs.AP
 // InspectCommitSetInTransaction performs the same job as InspectCommitSet
 // without the option of blocking for commits to finish so that it can run
 // inside an existing postgres transaction.  This is not an RPC.
-func (a *apiServer) InspectCommitSetInTransaction(txnCtx *txncontext.TransactionContext, commitset *pfs.CommitSet) ([]*pfs.CommitInfo, error) {
-	return a.driver.inspectCommitSetImmediate(txnCtx, commitset)
+func (a *apiServer) InspectCommitSetInTransaction(txnCtx *txncontext.TransactionContext, commitset *pfs.CommitSet, withAliases bool) ([]*pfs.CommitInfo, error) {
```

Comment:
Can we change the aliases parameter to `includeAliases` here and in the `inspectCommitSetImmediateTx` implementation. The negations aren't necessary.